### DISCUSSION TITLE
[Feature] Looks like function

### DISCUSSION THREAD

**Discussion by m4ntic0r** [immich-app/immich#10885] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

You upload a picture of a person temporarily and Immich tells you which person you have already recognized in your library most looks like the uploaded one.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9285

**Reply by m4ntic0r** [reply]
Its not that what i mean.

I know the functions included at the moment.

I tried to explain a function to upload a new person and immich compares that face to already known faces and show e.g. the 10 most look alike persons. Like a search function with an uploaded unknown face and immich shows people who look like that person.

why is that closed after 3 minutes? it has nothing to do with #9285. and i think even there the real feature request was not recognized.

**Reply by bo0tzz** [reply]
The feature request I linked is for exactly what you're describing here.

**Reply by m4ntic0r** [reply]
my fault. yes its the same from Author there. The answer from Collaborator was an other thing. And #9285 is still open so you´re right.
